ocaml/xapi/import.ml

```diff
@@ -2158,11 +2158,18 @@ let complete_import ~__context vmrefs =
         Xapi_vm_lifecycle.update_allowed_operations ~__context ~self:vm
       )
       vmrefs ;
-    (* We only keep VMs which are not snapshot *)
+    (* When only snapshots have been imported, return all of them.
```

Comment:
I don't have the context of this - but is this a behavior that customer expect?

Reply:
I was more confused by the difference between when only snapshots are imported and the case were a mix of snapshots and VMs are imported - as I read the comment, the behaviour is different.

Reply:
Before this PR the export of metadata-only was blocked, so I don't think this change of behaviour affects any users :)